incubator/hnc/pkg/reconcilers/object_test.go

```diff
@@ -44,13 +44,13 @@ var _ = Describe("Secret", func() {
 		setParent(ctx, barName, fooName)
 		setParent(ctx, bazName, barName)
 
-		Eventually(hasObject(ctx, "Role", barName, "foo-role")).Should(BeTrue())
+		Eventually(hasObject(ctx, "Role", barName, "foo-role"), 4*time.Second).Should(BeTrue())
```

Comment:
Save these magic frequencies somewhere as named constants

Reply:
Wait, why are we waiting this long? The object should show up right away, it only takes 3s for it to show up in the HNC Config status.

Reply:
Turns out we don't need to increase wait time at all after switching to the goroutine approach. I will resolve outdated test related comments for now.